config/base.json

```diff
@@ -10,7 +10,8 @@
 		},
 		"algolia": {
 			"docsIndexName": "product_DEVDOT",
-			"tutorialsIndexName": "prod_LEARN"
+			"tutorialsIndexName": "prod_LEARN",
+			"unifiedIndexName": "staging_DEVDOT_omni"
```

Comment:
I've added the `algolia.unifiedIndexName` property here to power unified search. I've set it to `staging_DEVDOT_omni` in all `env` _for now_. Intent is to add in `prod_DEVDOT_omni` later.